src/features/duplicates/hooks/useDuplicatesMutations.tsx

```diff
@@ -14,12 +14,12 @@ export default function useDuplicatesMutations(
   const dismissDuplicate = async (duplicateId: number) => {
     const dismissedDate = new Date().toISOString();
     await apiClient
-      .patch<ZetkinDuplicate>(
+      .patch<PotentialDuplicate>(
```

Comment:
I don't think this is a valid type for making a `PATCH` request with, because you can't patch all of the properties on `PotentialDuplicate` (only the `dismissed`, and only using a boolean). But that can be fixed in a future PR.

Reply:
I will fix it in my next PR!